queue.c

```diff
@@ -2741,6 +2748,9 @@ BaseType_t xQueueIsQueueFullFromISR( const QueueHandle_t xQueue )
                 mtCOVERAGE_TEST_MARKER();
             }
         }
+
+        /* Assert that the queue was added successfully */
+        configASSERT( ( ux != configQUEUE_REGISTRY_SIZE ) );

```

Comment:
I'm not totally sure if this should be added or not. It seems like it would be helpful to warn about failed vQueueAddToRegistry calls, but this might cause undesirable behavior. 

Currently, a user can add a given queue handle multiple times to the Queue Registry and vQueueAddToRegistry will not overwrite the existing entries for that queue handle.

Calls to pcQueueGetName always returns the first entry found for a given queue handle or NULL if the entry is not found.